packages/view-types/layer-controller/src/ImageAddButton.js

```diff
@@ -37,6 +37,7 @@ function ImageAddButton({ imageOptions, handleImageAdd }) {
       buttonIcon={<ImageAddIcon />}
       buttonClassName={classes.addButton}
       placement="bottom-start"
+      ariaLabel="Add image menu."
```

Comment:
Not a huge concern, but we should have consistency re: period usage in aria labels. I think we can probably drop the period since labels are meant to be short and concise; we can use `aria-description` for any longer descriptions that would require more than one sentence